### ISSUE TITLE
Allow CSV export of request tracking info

### ISSUE THREAD

**Issue by brockfanning** [usdoj/foia.gov#591]
In the backstage, on /admin/content/foia-requests, there needs to be a way to export the view results to a CSV file.

**Comment by gsolomon123** [comment]
@brockfanning Hi Brock, As expected, we are being asked about metrics re: requests submitted via the portal. I see this was assigned to Charles but he's out this week. Do you know what progress he's made? Thanks,
Gail

**Comment by brockfanning** [comment]
I'll check and get back to you.

**Comment by cnovick-doj** [comment]
Working on this right now.

**Comment by cnovick-doj** [comment]
@brockfanning, @gsolomon123, Some technical issues with CSV Export Module in D8: 

After installing the module I discovered that it looked to access the /sites/default/files directory but none was present (probably intentionally - as it may not be need for other tasks.) I created one manually on my local server and was able to get the site to download CSV files.

Mentioning this because deploying this this solution to a server will involve a similar process. 

Additionally, a query string needs to be added to the Data Export Path in the View configuration and currently there is a form which filters out query strings. See: https://www.drupal.org/project/views_data_export/issues/2957085

Issue has been very active over the last few days and the fix appears to be imminent.  Lets discuss a strategy when you are available.
